src/base/bittorrent/session.cpp

```diff
@@ -109,6 +114,10 @@ namespace
     void torrentQueuePositionTop(const libt::torrent_handle &handle);
     void torrentQueuePositionBottom(const libt::torrent_handle &handle);
 
+#ifdef Q_OS_WIN
+    QString convertIfaceNameToGuid(const QString &name);
```

Comment:
(quick comment only for this). Its already inside in an anonymous namespace.
And I don't think we will need it elsewhere.